src/client/modules/main/commands/focus/Focus.js

```diff
@@ -62,6 +64,9 @@ class Focus {
 								name: "focus color",
 								else: new ColorStep('color', {
 									errRequired: step => new Err('focus.colorRequired', "What focus color do you want?"),
+									else: new GradientStep('color', {
```

Comment:
Instead of using a new type of step, we can use `ColorStep` , making the first required:

```
else: [
	new ColorStep('color', {
		errRequired: step => new Err('focus.colorRequired', "What focus color do you want?"),
	}),
	new ColorStep('color2', {
		  errRequired: null,
	}),
],
```

Reply:
Though, that means we also need to, concatenate them when calling `addFocus` :

```javascript
this.module.charFocus.addFocus(char.id, c, params.color + (params.color2 || ''));
```